src/ui/webpack.config.js

```diff
@@ -169,13 +190,14 @@ module.exports = {
       }
     }
   },
+  mode: isDev === "1" ? "development" : "production",
```

Comment:
shouldn't it be `mode: isDev ? "development" : "production",` ?

Reply:
Yes, well spotted. We also need to the line where `isDev` is declared. It should be:
```
const isDev = process.argv.indexOf('-d') !== -1 || !!+process.env.BUILD_DEV;
```
Will fix this.